pytorch_lightning/utilities/apply_func.py

```diff
@@ -257,8 +258,11 @@ def move_data_to_device(batch: Any, device: Union[str, torch.device]) -> Any:
 
     def batch_to(data: Any) -> Any:
         # try to move torchtext data first
-        if _TORCHTEXT_AVAILABLE and isinstance(data, Batch):
-
+        if _TORCHTEXT_LEGACY and isinstance(data, Batch):
+            rank_zero_warn(
```

Comment:
We have this for deprecations

```suggestion
            rank_zero_deprecation(
```

Reply:
Done